template/api/article/models/article.settings.json

```diff
@@ -21,19 +24,6 @@
       "type": "richtext",
       "required": true
     },
-    "publishedAt": {
-      "type": "datetime",
-      "required": true
-    },
```

Comment:
I completely forgot about this. It's nice that we can remove it too

Reply:
agreed on the CLI change, it's so convenient.  Also updating the demo branch is a bit easier since I can create a project on that branch before this is merged and just drag the folders without ever leaving vscode.